pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -28,6 +28,10 @@
 
 # vCoolor dependency
 , gnome3
+
+# notational-fzf-vim dependencies
+, pypy3
```

Comment:
does is specifically need pypy3? and not python3? I would find this unlikely as pypy is not a mainstream interpreter

Reply:
Indeed, plugin docs says its optional.  Pulled pypy3 out as dependency and patched plugin to just use python3.